sources/src/configuration.ts

```diff
@@ -369,3 +369,15 @@ function getBooleanInput(paramName: string, paramDefault = false): boolean {
     }
     throw TypeError(`The value '${paramValue} is not valid for '${paramName}. Valid values are: [true, false]`)
 }
+
+function getMaybeBooleanInput(paramName: string): boolean | undefined {
```

Comment:
This should return a TypeError for any value besides '', 'false' or 'true'. I'll fix this.